text-command-line-terminal-applications-and-interfaces.md

```diff
@@ -1,8 +1,13 @@
 
-Though this class of applications predates the Web, WCAG can be applied to them. As noted in [Appendix B. Background on Text / Command-line / Terminal Applications and Interfaces](#background-on-text-command-line-terminal-applications-and-interfaces), applying WCAG to text / command-line applications involves understanding how text applications are rendered, how text applications have been made accessible via assistive technologies, and how to apply the concepts of “accessibility supported” and “programmatically determined” to text applications.
+With the exception of closed environments, assistive technology support in today's text applications differs markedly from that provided users in earlier days. The most marked difference is that there's likely less need to make a text-only version of an application accessible to AT, because there's almost certainly a GUI version which already supports the accessibility tree or platform accessibility API, and likely also a command line version which is highly comparable to the GUI version (such as that provided by Emacs). Key examples include:
+
+* Today's CLI support often includes context sensitive help so that help output following one command argument is different from the help provided following two arguments, and still different after three arguments. This helps users be more efficient and requires AT do nothing special beyond its core accessibility features.
+* Output options will generally include an option to write machine readable structured output (such as json) in addition to continuing reliance on still powerful, and widely used input/out redirection and piping options. In these scenarios the AT user is only relying on options used by anyone who finds the CLI environment compelling.
+ 
+Though these types of applications predate the Web, there has been a resurgence in text-based application interfaces, and WCAG can be applied to them. As noted in [Appendix B. Background on Text / Command-line / Terminal Applications and Interfaces](#background-on-text-command-line-terminal-applications-and-interfaces), applying WCAG to text / command-line applications involves understanding how text applications are rendered, how text applications have been made accessible via assistive technologies, and how to apply the concepts of “accessibility supported” and “programmatically determined” to text applications.
```

Comment:
The recent resurgence is already mentioned in a previous paragraph, so delete the first sentence here.

```suggestion
As noted in [Appendix B. Background on Text / Command-line / Terminal Applications and Interfaces](#background-on-text-command-line-terminal-applications-and-interfaces), applying WCAG to text / command-line applications involves understanding how text applications are rendered, how text applications have been made accessible via assistive technologies, and how to apply the concepts of “accessibility supported” and “programmatically determined” to text applications.
```